bigtable-dataflow-parent/bigtable-hbase-beam/src/main/java/com/google/cloud/bigtable/batch/common/CloudBigtableServiceImpl.java

```diff
@@ -29,8 +30,12 @@
   @Override
   public List<KeyOffset> getSampleRowKeys(CloudBigtableTableConfiguration config)
       throws IOException {
-    try (BigtableSession session = new BigtableSession(config.toBigtableOptions())) {
-      return session.getDataClientWrapper().sampleRowKeys(config.getTableId());
+
+    // TODO: figure out how to stick to HBase api here
```

Comment:
what does this mean exactly? Should we create a tracking issue?

Reply:
This would be part of making bigtable-hbase compatible with HBaseIO.  The work would need to be scoped and this would be a sub-task